adapters/postgres/postgres_test.go

```diff
@@ -704,6 +704,8 @@ func TestChkInvaidIdentifier(t *testing.T) {
 		{"SUM(test)", false},
 		{`SUM("test")`, false},
 		{"_123456789_123456789_123456789_123456789_123456789_123456789_12345", true},
+		{"not_invalid_table.with_not_invalid_column", false},
+		{"the_most_invalid_table_ever_seen_on_life_and_on_earth_at_least_today.the_most_invalid_column_ever_seen_on_life_and_on_earth_at_least_today", true},
```

Comment:
could you also add the case in the issue?

Reply:
Yes, I can!

Reply:
Done!

Reply:
@vmesel also would be good a case, maybe not here but where you do a test that uses a query similar to that mentioned by @manhofer in the issue.